e2e-tests/oauth-flow/run-tests.sh

```diff
@@ -2,7 +2,16 @@
 
 set -e # make script fail if any of the tests returns a non-zero exit code
 
-echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
-echo "!! Running test: OAuth flow       !!"
-echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+echo "!! Running test: OAuth flow (rfc002) !!"
```

Comment:
Maybe something like
```suggestion
echo "!! Running test: OAuth Bearer Access Token Flow (RFC002) !!"
```